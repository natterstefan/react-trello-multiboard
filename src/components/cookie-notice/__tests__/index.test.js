import React from 'react'
import { shallow } from 'enzyme'
import CookieNotice from '../'

describe('Component/CookieNotice', () => {
  test('should render without throwing an error', () => {
    expect(shallow(<CookieNotice />)).toMatchSnapshot()
  })
})
