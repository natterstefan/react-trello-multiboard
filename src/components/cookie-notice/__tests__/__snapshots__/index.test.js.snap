// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/CookieNotice should render without throwing an error 1`] = `
<div
  className="cookie-notice"
>
  <CookieBanner
    buttonMessage="OK"
    className="cookie-notice--banner"
    cookie="user-has-accepted-cookies"
    cookieExpiration={
      Object {
        "years": 1,
      }
    }
    cookiePath="/"
    cookies={
      Cookies {
        "HAS_DOCUMENT_COOKIE": true,
        "changeListeners": Array [],
        "cookies": Object {},
        "hooks": undefined,
      }
    }
    disableStyle={true}
    dismissOnScroll={false}
    dismissOnScrollThreshold={0}
    link={
      <Link
        replace={false}
        to="/privacy"
      >
        here.
      </Link>
    }
    message="We use cookies to give you the best online experience. See what we do and how you can disagree "
    onAccept={[Function]}
    styles={Object {}}
  />
</div>
`;
